tools/build/redo

```diff
@@ -274,12 +274,6 @@ Components = [
                   yaml = 'postdeploy.yml'),
 
     # the following (re)build images via gradle
-
-    makeComponent('actionproxy',
```

Comment:
i'd also keep this - as a convenience for building a runtime quickly and testing it all from this repo.